avm-transpiler/src/bit_traits.rs

```diff
@@ -65,19 +27,27 @@ impl BitsQueryable for MemoryAddress {
     }
 }
 
+macro_rules! impl_bits_queryable_for_integers {
```

Comment:
I've consulted with a few people and we think we prefer the explicit BitsQueryable implementations instead of the macro. The rest looks good and still valuable (if small).

Could you make this changes and merge master? Then I can duplicate the PR again, run CI and if it passes we merge.